lesson33-objects/index.js

```diff
@@ -0,0 +1,135 @@
+// MINIMUM
+/*
+ * 1. Створи об'єкт, що описує автомобіль (виробник, модель, рік випуску, середня швидкість,
+ * обсяг паливного баку, середня витрата палива на 100 км., водії), і наступні методи для
+ * роботи з цим об'єктом:
+ * 1.1 Метод, який виводить на екран інформацію про автомобіль.
+ * 1.2 Додавання ім’я водія у список
+ * 1.3 Перевірка водія на наявність його ім’я у списку
+ * 1.4 Підрахунок необхідного часу та кількості палива для подолання переданої відстані з
+ * середньою швидкістю. Враховуй, що через кожні 4 години дороги водієві необхідно робити перерву на 1 годину.
+ */
+
+const auto = {
+  brand: "Mazda",
+  model: 6,
+  year: 2022,
+  avgSpeed: 90,
+  fuelTank: 55,
+  fuelConsumption: 8,
+  drivers: ["Steve", "Kevin", "Georg"],
+  displayCarInfo: () => {
+    for (let key in auto) {
+      typeof auto[key] !== "function" ? console.log(`${key}: ${auto[key]}`) : null;
```

Comment:
be more precise with either using object or this because in method below you are using this